### PR TITLE
Remove yarn.lock from node_watcher and move it for dockerization

### PR DESCRIPTION
**Pull request by Tbaut**
- upgrade to 1.8.4 beta
- add @polkadot/api to package.json in node_watcher
- add a docker script to copy and remove yarn.lock form ../../ for dockerization.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -2471,6 +2487,24 @@
     eventemitter3 "^4.0.0"
     rxjs "^6.5.4"
 
+"@polkadot/api@1.8.0-beta.4", "@polkadot/api@^1.8.0-beta.4":
```
Comment (Tbaut) [inline yarn.lock]: Isn't that a problem to have `1.8.0-beta.4` and `^1.8.0-beta.4`?
I've had this in the past and couldn't figure out whether it was ok or not..

Comment (amaurymartiny) [reply yarn.lock]: No. The most important thing to have is one single `@polkadot/api` in yarn.lock

Comment (amaurymartiny) [reply yarn.lock]: Which is actually not the case now

**Comment by Tbaut** [comment]
ok, we now have docker scripts in `node_watcher` and `server`, local `yarn.lock` where removed. This is solution 1 from #245

**Pull request merged**